internal/model/label.go

```diff
@@ -27,7 +27,7 @@ var BetaLabel Label = Label{
 }
 
 var EnterpriseLabel Label = Label{
-	Name:        "Enterprise (E20)",
-	Description: "This plugin only works on servers with an E20 license.",
+	Name:        "Enterprise (E20 and Cloud)",
+	Description: "This plugin only works on servers with an E20 license or on Mattermost Cloud.",
```

Comment:
```suggestion
	Description: "This plugin only works on servers with an E20 license or with a Mattermost Cloud Instance.",
```
@justinegeffen do you know if "Instance" is how we are referring to a customer's "Cloud Server"?

Reply:
That's a very good question. Let me look into that.

Reply:
They're called workspaces; I've added my suggestion. :) 

Note on this: Does this work on Cloud Free, Professional, and Enterprise?